spec/system/support_interface/countries_spec.rb

```diff
@@ -159,6 +160,8 @@ def when_i_fill_teaching_authority_certificate
   end
 
   def when_i_fill_teaching_authority_other
+    fill_in "region-teaching-authority-other-field", with: "Other"
+  rescue Capybara::ElementNotFound
```

Comment:
Will this cause a default wait for the first case on line 163? I'm not sure, that might just be for js enabled tests. Would be good to avoid that if so.

Reply:
I don't believe so, at least this is a pattern I've used in a few other places in the tests and I haven't noticed any slowdowns, but I don't know for sure.